pkg/common/common.go

```diff
@@ -105,6 +112,19 @@ func SliceCopy[T any](s1 []T) []T {
 	return s2
 }
 
+func ReverseSlice[T any](input []T) []T {
```

Comment:
Where is it used?

Reply:
tests! But as it is quite generic, I decided to include in the common package. 

I can move thatto a test only code if you do not want code unused in "production"

Reply:
Sorry. I don't see it used. Not even in the tests.

Reply:
here https://github.com/Kuadrant/kuadrant-operator/pull/144/files#diff-b4522ffe746a4ae4e75451963ec5b9d946b1ea0c5524df85a3e897384c816b8cR75